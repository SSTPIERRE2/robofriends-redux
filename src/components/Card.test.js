import { shallow } from "enzyme";
import Card from "./Card";
import React from "react";

it("renders the Card UI", () => {
  expect(shallow(<Card />)).toMatchSnapshot();
});
