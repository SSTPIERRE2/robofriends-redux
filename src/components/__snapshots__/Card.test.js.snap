// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Card UI 1`] = `ShallowWrapper {}`;
